lib/postgrex/replication.ex

```diff
@@ -148,13 +142,17 @@ defmodule Postgrex.Replication do
   defstruct protocol: nil,
             state: nil,
             auto_reconnect: false,
-            reconnect_backoff: 500
+            reconnect_backoff: 500,
+            replication_started: false,
+            replication_opts: nil
```

Comment:
We don't need both `replication_started` and `replication_opts`. If `replication_opts` is nil, then replication has not started. :)